src/main/java/com/softserve/ita/java442/cityDonut/model/MediaType.java

```diff
@@ -0,0 +1,27 @@
+package com.softserve.ita.java442.cityDonut.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "media_type")
+public class MediaType {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name = "media_type", length = 5, nullable = false)
+    private String media_type;
+
+    @OneToMany(fetch = FetchType.EAGER, mappedBy = "mediaType")
```

Comment:
fetchType.Lazy

Reply:
In media entity FetchType is EAGER. Is this ok?